README.md

```diff
@@ -79,5 +79,56 @@ relDif  : 1.6464292361378058e-12)
 
 Complete API docs are available at [https://docs.balancer.finance/smart-contracts/api](https://docs.balancer.finance/smart-contracts/api)
 
+---
+
+## NEAR Protocol
+
+Balancer's Solidity contracts also work on the NEAR EVM. Visit https://near.org for details on the technology. The following steps will walk through how to set up a local NEAR node and run Balancer's Truffle tests.
+
+### Setup
```

Comment:
Given it's on betanet now - seems like instructions should just describe that and additionally link to near-web3-provider or something where we describe local development flow?

Reply:
thanks, added just the betanet directions. once there's an approved docs page on setting up a local EVM, I'll link to that. But you're right, we want to shepherd people to use betanet